### ISSUE TITLE
Just a question...

### ISSUE THREAD

**Issue by profzei** [johnnync13/Xiaomi-Mi-Air#77]
Sorry for my poor english and hi!
I don't have a Xiaomi-Mi-Air notebook but a similar one: Huawei Matebook X Pro 2018 with similar specs:
i7 8550u
Intel Bluetooth 8265
... and my question is about "your" IntelBluetoothFirmware.kext & IntelBluetoothInjector.kext.
Did you compile both of them in Mojave environment or Catalina one?
The version compiled in https://github.com/gnodipac886/MatebookXPro-hackintosh (which is my reference point for hackintosh) is for Catalina, but I'm still on Mohave 10.14.6 so it's not working properly...
The "basic" version is from https://github.com/zxystd/IntelBluetoothFirmware but it's for Catalina too. Only in the page for issue#7 (https://github.com/zxystd/IntelBluetoothFirmware/issues/7) there is a version of IntelBluetoothFirmware.kext for Mojave 10.14.6 but not for IntelBluetoothInjector.kext and I'm not sure if my Intel Bluetooth 8265 is "supported" in that version... so, since you have the same Intel Bluetooth 8265 I'm here asking you this...

Thank you for your reply

**Comment by radekwielonski** [comment]
Hey, I'm using this kext with Xiaomi Notebook Air 2017(i5-7200u) and it's working on Catalina(19C57). 

First I had problem with wake/sleep, but since I have disabled option "Allow Bluetooth Devices to wake this computer" in Bluetooth advanced settings, everything is working fine

Before that when I turned my computer into sleep mode I couldn't wake it up. After the reset in system report I've got error about something with IOUSBHostDevice.
